src/client/ui/Button.ts

```diff
@@ -0,0 +1,185 @@
+import * as PIXI from "pixi.js";
+import { ButtonState } from "./ButtonState.ts";
+import { gameConfig } from "../config/GameConfig.ts";
+
+export type stateUI = {
+  backgroundColor: number;
+  textLabel: string;
+  state: ButtonState;
+};
+
+export class Button extends PIXI.Container {
+  private background = new PIXI.Graphics();
+
+  private text = new PIXI.Text();
+
+  private _clickEvent: () => void;
```

Comment:
The "\_" prefix serves as a mark for private properties. Here you use the "private" keyword and "\_" simultaneously, while "background" and "text" doesn't have the same naming scheme, resulting in confusion.